src/platform/user/profile/vet360/components/base/Vet360EditModalActionButtons.jsx

```diff
@@ -63,7 +72,9 @@ class Vet360EditModalActionButtons extends React.Component {
   render() {
     const alertContent = (
       <div>
-        <h3>Are you sure?</h3>
+        <h3 tabIndex="-1" id="deleteConfirmationHeading">
```

Comment:
tabIndex was added to make the h3 focusable.  Text elements arent typically focusable.